frameworks/AutoGluon/requirements.txt

```diff
@@ -0,0 +1,3 @@
+mxnet
+scikit-learn==0.22.2
```

Comment:
@Innixma for some reason, I had to enforce scikit-learn version.
When I first updated autogluon version, everything was working fine, but after trying on a fresh environment, the process was hanging when loading autogluon.
Apparently, `scikit-optimize` doesn't support scikit-learn 0.23.0 yet, hence the issue. You may also want to enforce this in your setup.py as well.

Reply:
Yep, I realized a bit ago. Just updated autogluon to 0.0.9 to fix.

Reply:
done now, thx